docs/01_setup.md

```diff
@@ -83,31 +90,35 @@ Copy the config example and update the fields as necessary:
 cp deploy-config-example.json deploy-config.json
 ```
 
-There are already some deployed contracts that could be reused, copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save your gas fee.
+> [!IMPORTANT]
+> Make sure that you use the production zkeys (copy the correct generated coordinatorPubkey), set the `pollDuration` with the correct time in **seconds** and set `useQuadraticVoting` to false.
+
+### Deploy MACI Contracts
+
+There are already some deployed contracts that could be reused. Copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save gas fees.
 
 ```bash
 cp default-deployed-contracts.json deployed-contracts.json
 ```
 
-> [!IMPORTANT]
-> Make sure that you use the production zkeys, and set the `pollDuration` with the correct time on **seconds**.
-
-### Deploy MACI Contracts
-
-Run `pnpm deploy` to deploy the contracts (you can specify the network by appending `:network` to the command, e.g. pnpm deploy:sepolia - please refer to the available networks on the package.json scripts section)
-We highly recommend that if you already copy the `default-deployed-contracts.json` file to `deployed-contracts.json` file, run the following command to save your gas:
+We highly recommend that if you already copied the `default-deployed-contracts.json` file to `deployed-contracts.json` file, run the following command to save your gas:
 
 ```bash
 pnpm deploy:NETWORK --incremental
 ```
 
-Of course you could run without the `incremental` flag to deploy everything by yourself:
+Specify `NETWORK` with one of the available networks on the `package.json` scripts section.
+
+Of course, you can also deploy the contracts without the `incremental` flag:
+
+> [!WARNING]
+> If you decide to use this approach, make sure you have enough ETH, as deploying the full contracts suite will have a high transaction fee.
```

Comment:
is not necessarily high, it could quite cheap on op and other L2s, so maybe just say: [...]  as deploying the full contracts suite will incur in larger expenses.

Reply:
Done. This is mostly because my deployment failed at first because I only had 0.1 sepETH in my OP wallet. And it took me a while to figure out why it failed 😄 .

Reply:
haha np! thanks for sticking to it